### PR TITLE
Merge template v2.7.2

### PR DESCRIPTION
**Pull request by JoseEspinosa**
Merge template version 2.7.2

### CODE REVIEW

**Review by maxulysse — COMMENTED** [review]
Don't you miss some modules update?

**Comment by JoseEspinosa** [comment]
> Don't you miss some modules update?

Sorry, are you referring to the modules not included in the template?

**Comment by JoseEspinosa** [comment]
I think it should be good now @maxulysse

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Enhancements & fixes
 
+- Updated pipeline template to [nf-core/tools 2.7.2](https://github.com/nf-core/tools/releases/tag/2.7.2)
+
 ## [[2.0](https://github.com/nf-core/rnaseq/releases/tag/2.0)] - 2022-11-30
```
Comment (jfy133) [inline CHANGELOG.md:12]: RNAseq?

**Review by jfy133 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (JoseEspinosa) [reply CHANGELOG.md]: Good spot! seems a copy/paste error, fixed now

**Pull request merged**